utils/src/main/scala/za/co/absa/enceladus/utils/testUtils/HadoopFsTestBase.scala

```diff
@@ -13,12 +13,22 @@
  * limitations under the License.
  */
 
-package za.co.absa.enceladus
 
-import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider
+package za.co.absa.enceladus.utils.testUtils
 
-trait S3DefaultCredentialsProvider {
+import org.apache.hadoop.fs.FileSystem
+import org.apache.spark.sql.SparkSession
+import za.co.absa.enceladus.utils.fs.FileSystemUtils.FileSystemExt
 
-  implicit val defaultCredentialsProvider = DefaultCredentialsProvider.create()
 
+trait HadoopFsTestBase extends HasSparkSession {
+
+  implicit val fs = FileSystem.get(spark.sparkContext.hadoopConfiguration)
+  implicit val fsUtils = fs.toFsUtils
+}
+
+trait HasSparkSession {
```

Comment:
If you do not apply my `extends SparkTestBase` comment, I would still think of moving this to a separate file as it is used in the `SparkTestBase` the same way it is used in `HadoopFsTestBase`

Reply:
Sure, gladly. Done.